setup.py

```diff
@@ -28,7 +28,7 @@ def run_tests(self):
     'six>=1.5',
     'contextlib2; python_version=="2.7"',
     'mock; python_version=="2.7"',
-    'yarl; python_version>"3.5"',
+    'yarl; python_version>="3.5"',
```

Comment:
do we really need yarl as a vcrpy requirement? I ask because it's only used by aiohttp and it's already a requirement there [1]. @browniebroke would you please test if it's ok to remove it instead of changing the minimum version?

[1] https://github.com/aio-libs/aiohttp/blob/master/setup.py#L84

Reply:
There is a place in the codebase where `yarl` is imported and when this happens, usually the module should be listed as a dependency:

https://github.com/kevin1024/vcrpy/blob/a17624a4649308c2f5f9255391fad8ee4c0a351f/vcr/stubs/aiohttp_stubs/__init__.py#L10

Could this stub be applied if `aiohttp` isn't installed?

Reply:
I'm trying to remove it now, let's see if the build passes...

Reply:
@browniebroke I missed the line where yarl is explicitly used in the aiohttp stub. Therefore I agree with your initial statement: we should change the minimum version for yarl.

Reply:
No worries, I'll revert my last commit then :)